docs/UserGuide.md

```diff
@@ -66,95 +133,125 @@ adding, editing, deleting or viewing of friends in your gitGud friend list.
 
 Adds a new friend to gitGud friend’s list with an associated **unique** friend identifier.
 
+<div markdown="block" class="alert alert-info">
+**:information_source: What exactly is the friend identifier?**
+
+The friend identifier(FRIEND_ID) is a name you assign to a friend to refer to each unique friend for use in the gitGud application, 
```

Comment:
Consider using a code block for `FRIEND_ID` and including a space after the word identifier.